src/ManageCourses.Api/McConfig.cs

```diff
@@ -12,6 +12,7 @@ public class McConfig
         private const string ConfigKeyForPgServer = "MANAGE_COURSES_POSTGRESQL_SERVICE_HOST";
         private const string ConfigKeyForSignInUserInfoEndpoint = "auth:oidc:userinfo_endpoint";
         private const string ConfigKeyForApiKey = "api:key";
+        private const string ConfigKeyForBackendApiKey = "manage_courses_backend:key";
```

Comment:
with hindsight I'm not that keen on these nested things, what do you think?

Reply:
Should of went with the Options pattern